app/components/account/TokenAccountSection.tsx

```diff
@@ -497,3 +552,509 @@ function MultisigAccountCard({ account, info }: { account: Account; info: Multis
+                        <td className="text-lg-end">{extension.pendingBalanceCreditCounter}</td>
+                    </tr>
+                    <tr>
+                        <td>Expected Pending Balance Credit Counter</td>
+                        <td className="text-lg-end">{extension.expectedPendingBalanceCreditCounter}</td>
+                    </tr>
+                    <tr>
+                        <td>Actual Pending Balance Credit Counter</td>
+                        <td className="text-lg-end">{extension.actualPendingBalanceCreditCounter}</td>
+                    </tr>
+                    <tr>
+                        <td>Maximum Pending Balance Credit Counter</td>
+                        <td className="text-lg-end">{extension.maximumPendingBalanceCreditCounter}</td>
+                    </tr>
+                </>
+            );
+        }
+        case 'immutableOwner': {
+            return (
+                <tr>
+                    <td>Immutable Owner</td>
+                    <td className="text-lg-end">enabled</td>
+                </tr>
+            );
+        }
+        case 'memoTransfer': {
+            const extension = create(tokenExtension.state, MemoTransfer);
+            return (
+                <tr>
+                    <td>Require Memo on Incoming Transfers</td>
+                    <td className="text-lg-end">{extension.requireIncomingTransferMemos ? 'enabled' : 'disabled'}</td>
+                </tr>
+            );
+        }
+        case 'transferHookAccount': {
+            const extension = create(tokenExtension.state, TransferHookAccount);
+            return (
+                <tr>
+                    <td>Transfer Hook Account</td>
+                    <td className="text-lg-end">{!extension.transferring && 'not '}transferring</td>
+                </tr>
+            );
+        }
+        case 'nonTransferableAccount': {
+            return (
+                <tr>
+                    <td>Non-Transferable</td>
+                    <td className="text-lg-end">enabled</td>
+                </tr>
+            );
+        }
+        case 'confidentialTransferFeeAmount': {
+            const extension = create(tokenExtension.state, ConfidentialTransferFeeAmount);
+            return (
+                <tr>
+                    <td>Confidential Withheld Amount</td>
+                    <td className="text-lg-end">{extension.withheldAmount}</td>
+                </tr>
+            );
+        }
+        case 'tokenGroup': {
+            return (
+                <tr>
+                    <td>Group</td>
+                    <td className="text-lg-end">unparseable</td>
+                </tr>
+            );
+        }
+        case 'tokenGroupMember': {
+            return (
+                <tr>
+                    <td>Group Member</td>
+                    <td className="text-lg-end">unparseable</td>
+                </tr>
+            );
+        }
```

Comment:
How come this and `tokenGroup` are unparseable? The types in token-extension.ts look displayable

Reply:
Entirely out of laziness, done in 62f21de